### PR TITLE
reference: refine dmctl commands doc in DM

### PR DESCRIPTION
**Pull request by csuzhangxc**
### What is changed, added or deleted? 

- refine some dmctl commands doc
- add more dmctl commands doc

### Which version does your change affect? 

dev/v3.0/v2.1

### CODE REVIEW

**Comment by csuzhangxc** [comment]
@amyangfei @dcalvin PTAL

File: dev/reference/tools/data-migration/manage-tasks.md [hunk 1]
```diff
@@ -16,14 +16,17 @@ category: reference
...
 $ ./dmctl --help
 Usage of dmctl:
- # 打印版本信息
  -V prints version and exit
  # 按照 DM 提供的加密方法加密数据库密码，用于 DM 的配置文件
+ -config string
+       path to config file
  -encrypt string
        encrypt plaintext to ciphertext
  # DM-master 访问地址，dmctl 与 DM-master 交互以完成任务管理操作
```
Comment (3pointer) [inline dev/reference/tools/data-migration/manage-tasks.md]: 这里 DM-master 和下面的 dm-master 统一一下，
做个全局替换 DM-master 替换成 dm-master ？

File: dev/reference/tools/data-migration/manage-tasks.md [hunk 2]
```diff
@@ -480,7 +432,7 @@ stop-task [-w "127.0.0.1:10181"]  task-name
 
 2. 在 `task.yaml` 文件中更新需要修改的自定义配置或者错误配置。
 
-3. 使用 `start-task <task-name>` 恢复任务。
+3. 使用 `start-task <task-name>` 重启恢复任务。
```
Comment (3pointer) [inline dev/reference/tools/data-migration/manage-tasks.md]: 在这篇文章中也有两种写法: 
```
<task-name>  <task_name>
```
 这里也可以统一一下

File: dev/reference/tools/data-migration/manage-tasks.md (hunk 1 above)
Comment (csuzhangxc) [reply dev/reference/tools/data-migration/manage-tasks.md]: 这些没统一是由于直接复制的 dmctl 的输出，在文档里先改了 (in <https://github.com/pingcap/docs-cn/pull/1878/commits/dc191d3a6e70e08d4f709ac943350f03a739e369>)，后面我们再统一 dmctl 里的输出。

File: dev/reference/tools/data-migration/manage-tasks.md (hunk 2 above)
Comment (csuzhangxc) [reply dev/reference/tools/data-migration/manage-tasks.md]: done in <https://github.com/pingcap/docs-cn/pull/1878/commits/dc191d3a6e70e08d4f709ac943350f03a739e369>.

File: dev/reference/tools/data-migration/manage-tasks.md [hunk 3]
```diff
@@ -536,12 +488,282 @@ update-task [-w "127.0.0.1:10181"] ./task.yaml
...
+}
+```
+
+### 预设跳过 DDL 操作
+
+`sql-skip` 命令用于预设一个跳过操作，当 binlog event 的 position 或 SQL 语句与指定的 `binlog-pos` 或 `sql-pattern` 匹配时，执行该跳过操作。有关 `sql-skip` 的参数与结果解释，请参考[跳过 (skip) 或替代执行 (replace) 异常的 SQL 语句 文档中的 sql-skip](/dev/reference/tools/data-migration/skip-replace-sqls.md#sql-skip)。
+
+### 预设替代 DDL 操作
+
+`sql-replace` 命令用于预设一个替代执行操作，当 binlog event 的 position 或 SQL 语句与指定的 `binlog-pos` 或 `sql-pattern` 匹配时，执行该替代执行操作。有关 `sql-replace` 的参数与结果解释，请参考[跳过 (skip) 或替代执行 (replace) 异常的 SQL 语句 文档中的 sql-replace](/dev/reference/tools/data-migration/skip-replace-sqls.md#sql-replace)。
```
Comment (amyangfei) [inline dev/reference/tools/data-migration/manage-tasks.md]:
```suggestion
`sql-replace` 命令用于预设一个替代执行操作，当 binlog event 的 position 或 SQL 语句与指定的 `binlog-pos` 或 `sql-pattern` 匹配时，执行该替代执行操作。有关 `sql-replace` 的参数与结果解释，请参考[跳过 (skip) 或替代执行 (replace) 异常的 SQL 语句文档中的 sql-replace](/dev/reference/tools/data-migration/skip-replace-sqls.md#sql-replace)。
```

File: dev/reference/tools/data-migration/manage-tasks.md [hunk 4]
```diff
@@ -157,7 +157,7 @@ start-task [ -w "172.16.30.15:10081"] ./task.yaml
 query task's status
 
 Usage:
- dmctl query-status [-w worker ...] [task_name] [flags]
+ dmctl query-status [-w worker ...] [task-name] [flags]
```
Comment (amyangfei) [inline dev/reference/tools/data-migration/manage-tasks.md]: Currently the output is `dmctl query-status [-w worker ...] [task_name] [flags]` from `help query-status`, will update dmctl output later?

File: dev/reference/tools/data-migration/manage-tasks.md [hunk 5]
```diff
@@ -16,14 +16,17 @@ category: reference
 ```bash
 $ ./dmctl --help
 Usage of dmctl:
- # 打印版本信息
  -V prints version and exit
  # 按照 DM 提供的加密方法加密数据库密码，用于 DM 的配置文件
```
Comment (amyangfei) [inline dev/reference/tools/data-migration/manage-tasks.md]: This line should be before L23

File: dev/reference/tools/data-migration/manage-tasks.md (hunk 4 above)
Comment (csuzhangxc) [reply dev/reference/tools/data-migration/manage-tasks.md]: Do you think which of the following two is better:
1. keep them as the current output of dmctl
2. change them and update dmctl later

File: dev/reference/tools/data-migration/manage-tasks.md (hunk 5 above)
Comment (csuzhangxc) [reply dev/reference/tools/data-migration/manage-tasks.md]: fixed in https://github.com/pingcap/docs-cn/pull/1878/commits/e234e6af6dd8a31ac85a4b1578b67292e9b97ca3.

File: dev/reference/tools/data-migration/manage-tasks.md (hunk 4 above)
Comment (amyangfei) [reply dev/reference/tools/data-migration/manage-tasks.md]: I prefer 2, as we have already used `-` in command, we can keep them same

Comment (csuzhangxc) [reply dev/reference/tools/data-migration/manage-tasks.md]: I fire a PR in DM, https://github.com/pingcap/dm/pull/291.

File: dev/reference/tools/data-migration/manage-tasks.md [hunk 6]
```diff
@@ -453,13 +401,17 @@ stop-task [-w "127.0.0.1:10181"]  task-name
...
+- Table routing 规则
+- Black & white table lists 规则
+- Binlog event filter 规则
+- Column mapping 规则
 
 其余项均不支持更新。
 
+> **注意：**
+>
+> 如果能确保同步任务所需的 relay log 在任务停止期间不会被清理，则推荐使用[不支持更新项的更新步骤](#不支持更新项的更新步骤)来以统一的方式更新任务配置信息。
```
Comment (GregoryIan) [inline dev/reference/tools/data-migration/manage-tasks.md]: 这个应该在下个版本就会改正了吧

**Comment by csuzhangxc** [comment]
@dcalvin PTAL

File: dev/reference/tools/data-migration/manage-tasks.md [hunk 7]
```diff
@@ -624,12 +572,342 @@ Global Flags:
...
+}
+```
+
+### 预设跳过 DDL 操作
+
+`sql-skip` 命令用于预设一个跳过操作，当 binlog event 的 position 或 SQL 语句与指定的 `binlog-pos` 或 `sql-pattern` 匹配时，执行该跳过操作。有关 `sql-skip` 的参数与结果解释，请参考[跳过 (skip) 或替代执行 (replace) 异常的 SQL 语句 文档中的 sql-skip](/dev/reference/tools/data-migration/skip-replace-sqls.md#sql-skip)。
+
+### 预设替代 DDL 操作
+
+`sql-replace` 命令用于预设一个替代执行操作，当 binlog event 的 position 或 SQL 语句与指定的 `binlog-pos` 或 `sql-pattern` 匹配时，执行该替代执行操作。有关 `sql-replace` 的参数与结果解释，请参考[跳过 (skip) 或替代执行 (replace) 异常的 SQL 语句文档中的 sql-replace](/dev/reference/tools/data-migration/skip-replace-sqls.md#sql-replace)。
```
Comment (dcalvin) [inline dev/reference/tools/data-migration/manage-tasks.md]:
```suggestion
`sql-replace` 命令用于预设一个替换执行操作。当 binlog event 的 position 或 SQL 语句与指定的 `binlog-pos` 或 `sql-pattern` 匹配时，执行该替换执行操作。相关参数与结果解释，请参考[`sql-replace`](/dev/reference/tools/data-migration/skip-replace-sqls.md#sql-replace)。
```

File: dev/reference/tools/data-migration/manage-tasks.md [hunk 8]
```diff
@@ -624,12 +572,342 @@ Global Flags:
...
+        }
+    ]
+}
+```
+
+### 预设跳过 DDL 操作
+
+`sql-skip` 命令用于预设一个跳过操作，当 binlog event 的 position 或 SQL 语句与指定的 `binlog-pos` 或 `sql-pattern` 匹配时，执行该跳过操作。有关 `sql-skip` 的参数与结果解释，请参考[跳过 (skip) 或替代执行 (replace) 异常的 SQL 语句 文档中的 sql-skip](/dev/reference/tools/data-migration/skip-replace-sqls.md#sql-skip)。
+
+### 预设替代 DDL 操作
```
Comment (dcalvin) [inline dev/reference/tools/data-migration/manage-tasks.md]:
```suggestion
### 预设替换 DDL 操作
```

File: dev/reference/tools/data-migration/manage-tasks.md [hunk 9]
```diff
@@ -624,12 +572,342 @@ Global Flags:
...
+            "worker": "127.0.0.1:8262",
+            "msg": ""
+        }
+    ]
+}
+```
+
+### 预设跳过 DDL 操作
+
+`sql-skip` 命令用于预设一个跳过操作，当 binlog event 的 position 或 SQL 语句与指定的 `binlog-pos` 或 `sql-pattern` 匹配时，执行该跳过操作。有关 `sql-skip` 的参数与结果解释，请参考[跳过 (skip) 或替代执行 (replace) 异常的 SQL 语句 文档中的 sql-skip](/dev/reference/tools/data-migration/skip-replace-sqls.md#sql-skip)。
```
Comment (dcalvin) [inline dev/reference/tools/data-migration/manage-tasks.md]:
```suggestion
`sql-skip` 命令用于预设一个跳过操作。当 binlog event 的 position 或 SQL 语句与指定的 `binlog-pos` 或 `sql-pattern` 匹配时，执行该跳过操作。相关参数与结果解释，请参考[`sql-skip`](/dev/reference/tools/data-migration/skip-replace-sqls.md#sql-skip)。
```

File: dev/reference/tools/data-migration/manage-tasks.md [hunk 10]
```diff
@@ -70,43 +74,40 @@ Usage:
...
+  migrate-relay        migrate dm-worker's relay unit
+  pause-relay          pause dm-worker's relay unit
   pause-task           pause a running task with name
+  purge-relay          purge dm-worker's relay log files according to specified filename
+  query-error          query task's error
   query-status         query task's status
   refresh-worker-tasks refresh worker -> tasks mapper
-  resume-relay         resume DM-worker's relay unit
+  resume-relay         resume dm-worker's relay unit
   resume-task          resume a paused task with name
```
Comment (dcalvin) [inline dev/reference/tools/data-migration/manage-tasks.md]:
```suggestion
  resume-task          resume a specified paused task
```

File: dev/reference/tools/data-migration/manage-tasks.md [hunk 11]
```diff
@@ -624,12 +572,342 @@ Global Flags:
...
+```
+{
+    "result": true,
+    "msg": "check pass!!!"
+}
+```
+
+### 暂停 relay 处理单元
+
+relay 处理单元在 DM-worker 进程启动后即开始自动运行，通过使用 `pause-relay` 命令，我们可以暂停 relay 处理单元的运行。
```
Comment (dcalvin) [inline dev/reference/tools/data-migration/manage-tasks.md]:
```suggestion
relay 处理单元在 DM-worker 进程启动后即开始自动运行。通过使用 `pause-relay` 命令，我们可以暂停 relay 处理单元的运行。
```

File: dev/reference/tools/data-migration/manage-tasks.md [hunk 12]
```diff
@@ -624,12 +572,342 @@ Global Flags:
...
+在使用 `start-task` 启动同步任务时，DM 也会做 `check-task` 所做的全部检查。
+
+{{< copyable "" >}}
+
+```bash
+help check-task
+```
+
+```
+check a task with config file
```
Comment (dcalvin) [inline dev/reference/tools/data-migration/manage-tasks.md]:
```suggestion
check the config file of the task
```

File: dev/reference/tools/data-migration/manage-tasks.md [hunk 13]
```diff
@@ -624,12 +572,342 @@ Global Flags:
...
+            "worker": "172.16.30.15:8262",
+            "msg": ""
+        }
+    ]
+}
+```
+
+### 切换 relay log 到新的子目录
+
+relay 处理单元通过使用不同的子目录来存储来自上游不同 MySQL 实例的 binlog 数据，通过使用 `switch-relay-master` 命令我们可以变更 relay 处理单元以开始使用一个新的子目录。
```
Comment (dcalvin) [inline dev/reference/tools/data-migration/manage-tasks.md]:
```suggestion
relay 处理单元通过使用不同的子目录来存储来自上游不同 MySQL 实例的 binlog 数据。通过使用 `switch-relay-master` 命令，我们可以变更 relay 处理单元以开始使用一个新的子目录。
```

File: dev/reference/tools/data-migration/manage-tasks.md [hunk 14]
```diff
@@ -624,12 +572,342 @@ Global Flags:
...
+
+```bash
+switch-relay-master -w "127.0.0.1:8262"
+```
+
+#### 参数解释
+
+- `-w`：
+    - 必选
+    - 指定在特定的一个 DM-worker 上切换 relay 处理单元所使用的子目录
```
Comment (dcalvin) [inline dev/reference/tools/data-migration/manage-tasks.md]:
```suggestion
    - 指定需要切换 relay 处理单元使用子目录的 DM-worker 
```

File: dev/reference/tools/data-migration/manage-tasks.md [hunk 15]
```diff
@@ -624,12 +572,342 @@ Global Flags:
...
+
+```bash
+resume-relay -w "127.0.0.1:8262"
+```
+
+#### 参数解释
+
+- `-w`：
+    - 必选
+    - 指定在特定的一个 DM-worker 上恢复 relay 处理单元
```
Comment (dcalvin) [inline dev/reference/tools/data-migration/manage-tasks.md]:
```suggestion
    - 指定需要恢复 relay 处理单元的 DM-worker 
```

File: dev/reference/tools/data-migration/manage-tasks.md [hunk 16]
```diff
@@ -624,12 +572,342 @@ Global Flags:
...
+
+```bash
+pause-relay -w "127.0.0.1:8262"
+```
+
+#### 参数解释
+
+- `-w`：
+    - 必选
+    - 指定在特定的一个 DM-worker 上暂停 relay 处理单元
```
Comment (dcalvin) [inline dev/reference/tools/data-migration/manage-tasks.md]:
```suggestion
    - 指定需要暂停 relay 处理单元的 DM-worker
```

File: dev/reference/tools/data-migration/manage-tasks.md [hunk 17]
```diff
@@ -624,12 +572,342 @@ Global Flags:
...
+
+```bash
+purge-relay -w "127.0.0.1:8262" --filename "mysql-bin.000003"
+```
+
+#### 参数解释
+
+- `-w`：
+    - 必选
+    - 指定在特定的一个 DM-worker 上执行 relay log 清理操作
```
Comment (dcalvin) [inline dev/reference/tools/data-migration/manage-tasks.md]:
```suggestion
    - 指定需要执行 relay log 清理操作的 DM-worker
```

File: dev/reference/tools/data-migration/manage-tasks.md [hunk 18]
```diff
@@ -624,12 +572,342 @@ Global Flags:
...
+```
+
+```
+purge dm-worker's relay log files according to specified filename
+
+Usage:
+  dmctl purge-relay <-w worker> [--filename] [--sub-dir] [flags]
+
+Flags:
+  -f, --filename string   whether try to purge relay log files before this filename, the format is "mysql-bin.000006"
```
Comment (dcalvin) [inline dev/reference/tools/data-migration/manage-tasks.md]:
```suggestion
  -f, --filename string   name of the terminal file before which to purge relay log files. Sample format: "mysql-bin.000006"
```

File: dev/reference/tools/data-migration/manage-tasks.md [hunk 19]
```diff
@@ -624,12 +572,342 @@ Global Flags:
...
+  -h, --help              help for purge-relay
+  -s, --sub-dir string    specify relay sub directory for --filename, if not specified, the latest one will be used, the format is "2ae76434-f79f-11e8-bde2-0242ac130008.000001"
```
Comment (dcalvin) [inline dev/reference/tools/data-migration/manage-tasks.md]:
```suggestion
  -s, --sub-dir string    specify relay sub directory for --filename. If not specified, the latest one will be used. Sample format: "2ae76434-f79f-11e8-bde2-0242ac130008.000001"
```

File: dev/reference/tools/data-migration/manage-tasks.md [hunk 20]
```diff
@@ -624,12 +572,342 @@ Global Flags:
...
+```
+
+#### 参数解释
+
+- `-w`：
+    - 必选
+    - 指定在特定的一个 DM-worker 上执行 relay log 清理操作
+- `--filename`：
+    - 必选
+    - 指定 relay log 将要停止清理的文件名，如指定为 `mysql-bin.000100`，则只尝试清理到 `mysql-bin.000099`
```
Comment (dcalvin) [inline dev/reference/tools/data-migration/manage-tasks.md]:
```suggestion
    - 指定标识 relay log 将要停止清理的文件名。如指定为 `mysql-bin.000100`，则只尝试清理到 `mysql-bin.000099`
```

File: dev/reference/tools/data-migration/manage-tasks.md [hunk 21]
```diff
@@ -624,12 +572,342 @@ Global Flags:
...
+#### 返回结果示例
+
+{{< copyable "" >}}
+
+```bash
+purge-relay -w "127.0.0.1:8262" --filename "mysql-bin.000003"
+```
+
+```
+[warn] no --sub-dir specify for --filename, the latest one will be used
```
Comment (dcalvin) [inline dev/reference/tools/data-migration/manage-tasks.md]:
```suggestion
[warn] no --sub-dir specified for --filename; the latest one will be used
```

**Comment by csuzhangxc** [comment]
@dcalvin thanks a lot! PTAL again

File: dev/reference/tools/data-migration/manage-tasks.md [hunk 22]
```diff
@@ -624,12 +572,342 @@ Global Flags:
...
+            "worker": "172.16.30.15:8262",
+            "msg": ""
+        }
+    ]
+}
+```
+
+### 切换 relay log 到新的子目录
+
+relay 处理单元通过使用不同的子目录来存储来自上游不同 MySQL 实例的 binlog 数据，通过使用 `switch-relay-master` 命令，我们可以变更 relay 处理单元以开始使用一个新的子目录。
```
Comment (dcalvin) [inline dev/reference/tools/data-migration/manage-tasks.md]:
```suggestion
relay 处理单元通过使用不同的子目录来存储来自上游不同 MySQL 实例的 binlog 数据。通过使用 `switch-relay-master` 命令，我们可以变更 relay 处理单元以开始使用一个新的子目录。
```

File: dev/reference/tools/data-migration/manage-tasks.md [hunk 23]
```diff
@@ -70,25 +74,29 @@ Usage:
...
+  sql-skip             skip the binlog event matched by a specific binlog position (binlog-pos) or SQL pattern (sql-pattern)
   start-task           start a task with config file
-  stop-task            stop a task with name
-  switch-relay-master  switch master server of DM-worker's relay unit
-  unlock-ddl-lock      force to unlock DDL lock
-  update-master-config update configure of DM-master
-  update-task          update a task's config for routes, filters, column-mappings, black-white-list
+  stop-task            stop a specified task
+  switch-relay-master  switch the master server of the DM-worker's relay unit
+  unlock-ddl-lock      forcefully to unlock DDL lock
```
Comment (dcalvin) [inline dev/reference/tools/data-migration/manage-tasks.md]:
```suggestion
  unlock-ddl-lock      forcefully unlock DDL lock
```

File: dev/reference/tools/data-migration/manage-tasks.md [hunk 24]
```diff
@@ -70,25 +74,29 @@ Usage:
...
   resume-relay         resume DM-worker's relay unit
-  resume-task          resume a paused task with name
+  resume-task          resume a specified paused task
   show-ddl-locks       show un-resolved DDL locks
   sql-inject           sql-inject injects (limited) sqls into syncer as binlog event
-  sql-replace          sql-replace replaces sql in specific binlog_pos with other sqls, each sql must ends with semicolon;
-  sql-skip             sql-skip skips specified binlog position
+  sql-replace          replace SQLs matched by a specific binlog position (binlog-pos) or a SQL pattern (sql-pattern); each SQL must end with a semicolon
+  sql-skip             skip the binlog event matched by a specific binlog position (binlog-pos) or SQL pattern (sql-pattern)
   start-task           start a task with config file
```
Comment (dcalvin) [inline dev/reference/tools/data-migration/manage-tasks.md]:
```suggestion
  start-task           start a task as defined in the config file
```

File: dev/reference/tools/data-migration/manage-tasks.md [hunk 25]
```diff
@@ -70,25 +74,29 @@ Usage:
...
   refresh-worker-tasks refresh worker -> tasks mapper
   resume-relay         resume DM-worker's relay unit
-  resume-task          resume a paused task with name
+  resume-task          resume a specified paused task
   show-ddl-locks       show un-resolved DDL locks
   sql-inject           sql-inject injects (limited) sqls into syncer as binlog event
-  sql-replace          sql-replace replaces sql in specific binlog_pos with other sqls, each sql must ends with semicolon;
-  sql-skip             sql-skip skips specified binlog position
+  sql-replace          replace SQLs matched by a specific binlog position (binlog-pos) or a SQL pattern (sql-pattern); each SQL must end with a semicolon
+  sql-skip             skip the binlog event matched by a specific binlog position (binlog-pos) or SQL pattern (sql-pattern)
```
Comment (dcalvin) [inline dev/reference/tools/data-migration/manage-tasks.md]:
```suggestion
  sql-skip             skip the binlog event matched by a specific binlog position (binlog-pos) or a SQL pattern (sql-pattern)
```

File: dev/reference/tools/data-migration/manage-tasks.md [hunk 26]
```diff
@@ -70,25 +74,29 @@ Usage:
...
+  pause-task           pause a specified running task
+  purge-relay          purge relay log files of the DM-worker according to the specified filename
+  query-error          query task error
+  query-status         query task status
   refresh-worker-tasks refresh worker -> tasks mapper
   resume-relay         resume DM-worker's relay unit
-  resume-task          resume a paused task with name
+  resume-task          resume a specified paused task
   show-ddl-locks       show un-resolved DDL locks
   sql-inject           sql-inject injects (limited) sqls into syncer as binlog event
```
Comment (dcalvin) [inline dev/reference/tools/data-migration/manage-tasks.md]:
```suggestion
  sql-inject            inject (limited) SQLs into Syncer as binlog events
```

File: dev/reference/tools/data-migration/manage-tasks.md [hunk 27]
```diff
@@ -70,25 +74,29 @@ Usage:
   dmctl [command]
 
 Available Commands:
-  break-ddl-lock       force to break DM-worker's DDL lock
-  generate-task-config generate a task config with config file
+  break-ddl-lock       forcefully break DM-worker's DDL lock
+  check-task           check the config file of the task
   help                 Help about any command
```
Comment (dcalvin) [inline dev/reference/tools/data-migration/manage-tasks.md]:
```suggestion
  help                 help about any command
```

File: dev/reference/tools/data-migration/manage-tasks.md [hunk 28]
```diff
@@ -624,12 +572,342 @@ Global Flags:
...
+
+## 其他任务与集群管理命令
+
+除上述常用的任务管理命令外，DM 还提供了其他一些命令用于管理数据同步任务或 DM 集群本身。
+
+### 检查任务配置文件
+
+`check-task` 命令用于检查指定的数据同步任务配置文件（`task.yaml`）是否合法以及上下游数据库的配置、权限、表结构等是否满足同步需要，具体可参考[上游 MySQL 实例配置前置检查](/dev/reference/tools/data-migration/precheck.md)。
+
+在使用 `start-task` 启动同步任务时，DM 也会做 `check-task` 所做的全部检查。
```
Comment (dcalvin) [inline dev/reference/tools/data-migration/manage-tasks.md]:
```suggestion
在使用 `start-task` 启动同步任务时，DM 也会执行 `check-task` 所做的全部检查。
```

File: dev/reference/tools/data-migration/manage-tasks.md [hunk 29]
```diff
@@ -624,12 +572,342 @@ Global Flags:
...
-详见[手动处理 sharding DDL lock](/dev/reference/tools/data-migration/features/manually-handling-sharding-ddl-locks.md)。
+目前与 DDL lock 相关的命令主要包括 `show-ddl-locks`、`unlock-ddl-lock`、`break-ddl-lock` 等。有关它们的功能、用法以及适用场景等，请参考[手动处理 sharding DDL lock](/dev/reference/tools/data-migration/features/manually-handling-sharding-ddl-locks.md)。
+
+## 其他任务与集群管理命令
+
+除上述常用的任务管理命令外，DM 还提供了其他一些命令用于管理数据同步任务或 DM 集群本身。
+
+### 检查任务配置文件
+
+`check-task` 命令用于检查指定的数据同步任务配置文件（`task.yaml`）是否合法以及上下游数据库的配置、权限、表结构等是否满足同步需要，具体可参考[上游 MySQL 实例配置前置检查](/dev/reference/tools/data-migration/precheck.md)。
```
Comment (dcalvin) [inline dev/reference/tools/data-migration/manage-tasks.md]:
```suggestion
`check-task` 命令用于检查指定的数据同步任务配置文件（`task.yaml`）是否合法以及上下游数据库的配置、权限、表结构等是否满足同步需要。具体可参考[上游 MySQL 实例配置前置检查](/dev/reference/tools/data-migration/precheck.md)。
```

File: dev/reference/tools/data-migration/manage-tasks.md [hunk 30]
```diff
@@ -624,12 +572,342 @@ Global Flags:
...
+    "result": true,
+    "msg": "check pass!!!"
+}
+```
+
+### 暂停 relay 处理单元
+
+relay 处理单元在 DM-worker 进程启动后即开始自动运行。通过使用 `pause-relay` 命令，我们可以暂停 relay 处理单元的运行。
+
+当需要切换 DM-worker 通过虚拟 IP 连接的上游 MySQL 时，我们需要使用 `pause-relay` 对 DM 执行变更，具体变更步骤请参考[虚拟 IP 环境下的上游主从切换](/dev/reference/tools/data-migration/usage-scenarios/master-slave-switch.md#虚拟-IP-环境下的上游主从切换)。
```
Comment (dcalvin) [inline dev/reference/tools/data-migration/manage-tasks.md]:
```suggestion
当需要切换 DM-worker 通过虚拟 IP 连接的上游 MySQL 时，我们需要使用 `pause-relay` 对 DM 执行变更。具体变更步骤请参考[虚拟 IP 环境下的上游主从切换](/dev/reference/tools/data-migration/usage-scenarios/master-slave-switch.md#虚拟-IP-环境下的上游主从切换)。
```

File: dev/reference/tools/data-migration/manage-tasks.md [hunk 31]
```diff
@@ -624,12 +572,342 @@ Global Flags:
...
+        }
+    ]
+}
+```
+
+### 恢复 relay 处理单元
+
+`resume-relay` 用于恢复处于 `Paused` 状态的 relay 处理单元。
+
+当需要切换 DM-worker 通过虚拟 IP 连接的上游 MySQL 时，我们需要使用 `resume-relay` 对 DM 执行变更，具体变更步骤请参考[虚拟 IP 环境下的上游主从切换](/dev/reference/tools/data-migration/usage-scenarios/master-slave-switch.md#虚拟-IP-环境下的上游主从切换)。
```
Comment (dcalvin) [inline dev/reference/tools/data-migration/manage-tasks.md]:
```suggestion
当需要切换 DM-worker 通过虚拟 IP 连接的上游 MySQL 时，我们需要使用 `resume-relay` 对 DM 执行变更。具体变更步骤请参考[虚拟 IP 环境下的上游主从切换](/dev/reference/tools/data-migration/usage-scenarios/master-slave-switch.md#虚拟-IP-环境下的上游主从切换)。
```

File: dev/reference/tools/data-migration/manage-tasks.md [hunk 32]
```diff
@@ -624,12 +572,342 @@ Global Flags:
...
+        }
+    ]
+}
+```
+
+### 切换 relay log 到新的子目录
+
+relay 处理单元通过使用不同的子目录来存储来自上游不同 MySQL 实例的 binlog 数据，通过使用 `switch-relay-master` 命令，我们可以变更 relay 处理单元以开始使用一个新的子目录。
+
+当需要切换 DM-worker 通过虚拟 IP 连接的上游 MySQL 时，我们需要使用 `switch-relay-master` 对 DM 执行变更，具体变更步骤请参考[虚拟 IP 环境下的上游主从切换](/dev/reference/tools/data-migration/usage-scenarios/master-slave-switch.md#虚拟-IP-环境下的上游主从切换)。
```
Comment (dcalvin) [inline dev/reference/tools/data-migration/manage-tasks.md]:
```suggestion
当需要切换 DM-worker 通过虚拟 IP 连接的上游 MySQL 时，我们需要使用 `switch-relay-master` 对 DM 执行变更。具体变更步骤请参考[虚拟 IP 环境下的上游主从切换](/dev/reference/tools/data-migration/usage-scenarios/master-slave-switch.md#虚拟-IP-环境下的上游主从切换)。
```

File: dev/reference/tools/data-migration/manage-tasks.md [hunk 33]
```diff
@@ -221,110 +222,35 @@ Global Flags:
 
 有关查询结果中各参数的意义，详见[查询状态结果](/dev/reference/tools/data-migration/query-status.md#查询结果)。
```
Comment (dcalvin) [inline dev/reference/tools/data-migration/manage-tasks.md]: where is the sample result here? @csuzhangxc

Comment (csuzhangxc) [reply dev/reference/tools/data-migration/manage-tasks.md]: https://github.com/pingcap/docs-cn/blob/master/dev/reference/tools/data-migration/query-status.md#%E6%9F%A5%E8%AF%A2%E7%BB%93%E6%9E%9C has a complete result.

File: dev/reference/tools/data-migration/manage-tasks.md (hunk 2 above)
Comment (dcalvin) [reply dev/reference/tools/data-migration/manage-tasks.md]: In docs, there is step 3 "3. Update the task configuration using `update-task task.yaml`." Why is it missing here? @csuzhangxc

**Pull request merged**